importer/UsEn.py

```diff
@@ -0,0 +1,110 @@
+import dateparser
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class UsEn(Monument):
+
+    def set_location(self):
+        """Set Location based on linked article."""
+        if self.has_non_empty_attribute("city"):
+            if utils.count_wikilinks(self.city) == 1:
+                loc_q = utils.q_from_first_wikilink("en", self.city)
+                self.add_statement("location", loc_q)
+            else:
+                self.add_to_report("city", self.city, "location")
+
+    def set_adm_location(self):
+        """
+        Set administrative location.
+
+        First, try using linked County matched against
+        external list.
+        If that doesn't yield, use ISO code of State
+        against external list.
+        """
+        adm_q = None
+        county_dic = self.data_files["counties"]
+        state_dic = self.data_files["states"]
+
+        if utils.count_wikilinks(self.county) == 1:
+            county_q = utils.q_from_first_wikilink("en", self.county)
+            if county_q in [x["item"] for x in county_dic]:
+                adm_q = county_q
```

Comment:
add to report already here. We care about a failed match even if we got a conciliatory state match